DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -144,6 +144,7 @@ private function sessionLoad($config, ContainerBuilder $container)
                         $loaded['jackalope'] = true;
                     }
                     $this->loadJackalopeSession($session, $container, $type);
+                    $this->loadJackalopeSession($session, $container, $type, true);
```

Comment:
was this just forgotten in the original PR? reading through the code i dont see that the `$admin` parameter was ever used without this change so it would have been a dead parameter. seems like. the loadJackalopeSession has all kinds of logic to build the admin session.